install/index.md

```diff
@@ -27,48 +43,20 @@ apps. Docker Engine - Community has three types of update channels, **stable**,
 * **Test** gives pre-releases that are ready for testing before general availability.
 * **Nightly** gives you latest builds of work in progress for the next major release.
 
-## Releases
-
-For the Docker Engine - Community, the open
-repositories [Docker Engine](https://github.com/docker/engine) and
-[Docker Client](https://github.com/docker/cli) apply.
-
-Releases of Docker Engine and Docker Client for general availability
-are versioned using dotted triples. The components of this triple
-are `YY.mm.<patch>` where the `YY.mm` component is referred to as the
-year-month release. The version numbering format is chosen to illustrate
-cadence and does not guarantee SemVer, but the desired date for general
-availability. The version number may have additional information, such as
-beta and release candidate qualifications. Such releases are considered
-"pre-releases".
-
-The cadence of the year-month releases is every 6 months starting with
-the `18.09` release. The patch releases for a year-month release take
-place as needed to address bug fixes during its support cycle.
-
-Docker Engine - Community binaries for a release are available on [download.docker.com](https://download.docker.com/)
-as packages for the supported operating systems. Docker Engine - Enterprise binaries are
-available on the [Docker Hub](https://hub.docker.com/) for the supported operating systems. The
-release channels are available for each of the year-month releases and
-allow users to "pin" on a year-month release of choice. The release
-channel also receives patch releases when they become available.
-
-### Nightly builds
+### Stable
```

Comment:
The description of the channels is quite verbose/detailed (and may not be of interest to most people). Thinking if we should move the "Supported platforms" and "Support" headings up (above the detailed descriptions of the channels).


Something like:

- (introduction)
- Releases (perhaps rename to "release channels"?)
- Supported platforms
- Reporting security issues
- Support

The "exceptions" heading is a bit confusing now (as there's no description of the release cadence); perhaps we should remove it altogether

Reply:
Moved the details to a new section "Release channels" and added a cross-reference to it from the overview section. Thought it might be appropriate to move Support after Release channels. Here's what it looks like now:

- Overview 
- Supported platforms
- Release channels
- Support
- Reporting security issues
- Get started